.editorconfig

```diff
@@ -2,10 +2,11 @@
 
 root = true
 
-[*.{js,ts,hbs,json}]
+[*]
 
 # Change these settings to your own preference
-indent_style = tab
```

Comment:
Personally I like tabs more then spaces but if everybody is up for this I guess I'll just have to get used to this. Another thing that might be good is to let everybody (including the guys from webGL and games) know about these new coding standards. 

A lot of times when I work with them I get the feeling they are not involved with the standards and then when we implement their code we end up with an unfixable amount of linting issues which usually leads to disabling several rules or disabling it entirely

Reply:
+1 on the coding standards